src/wasm-emscripten.h

```diff
@@ -40,6 +40,9 @@ class EmscriptenGlueGenerator {
   // signature in the indirect function table.
   void generateDynCallThunks();
 
+  // Fix up EM_ASM calls.
```

Comment:
It would be good to get into more details here. Why do they need to be fixed up?